lib/stdlib/src/timer.erl

```diff
@@ -358,8 +358,8 @@ init([]) ->
 
 %% server calls
 
-% Try sending a call. If it fails with reason noproc,
-% try starting the timer server and try once again.
+                                                % Try sending a call. If it fails with reason noproc,
+                                                % try starting the timer server and try once again.
```

Comment:
That one's probably not right. Emacs likely wants `%%` instead of `%`.

Reply:
Keen eyes :) Thanks for pointing it out, I'll change.